File: Java/Program/Toko.java
public class Toko extends Perusahaan {
    private String namaToko;
    private double penghasilan;
    private String jamOperasional;

    public Toko(String NIB, String namaPemilik, String lokasi,
                String namaToko, double penghasilan, String jamOperasional) {
        super(NIB, namaPemilik, lokasi);
        this.namaToko = namaToko;
        this.penghasilan = penghasilan;
        this.jamOperasional = jamOperasional;
    }

    public Toko() {
        this("", "", "", "", 0.0, "");
    }

    public String getNamaToko() { return namaToko; }
    public void setNamaToko(String namaToko) { this.namaToko = namaToko; }

    public double getPenghasilan() { return penghasilan; }
    public void setPenghasilan(double penghasilan) { this.penghasilan = penghasilan; }

    public String getJamOperasional() { return jamOperasional; }
    public void setJamOperasional(String jamOperasional) { this.jamOperasional = jamOperasional; }
}

File: Python/Program/Toko.py
from Perusahaan import Perusahaan

class Toko(Perusahaan):
    def __init__(self, NIB="", pemilik="", lokasi="", namaToko="", penghasilan=0.0, jamOperasional=""):
        super().__init__(NIB, pemilik, lokasi)
        self.namaToko = namaToko
        self.penghasilan = penghasilan
        self.jamOperasional = jamOperasional

    # Getter
    def getNamaToko(self): return self.namaToko
    def getPenghasilan(self): return self.penghasilan
    def getJamOperasional(self): return self.jamOperasional

    # Setter
    def setNamaToko(self, nama): self.namaToko = nama
    def setPenghasilan(self, penghasilan): self.penghasilan = penghasilan
    def setJamOperasional(self, jam): self.jamOperasional = jam

    def tampilkanTabel(self):
        print("Toko umum:", self.namaToko)
